aten/src/ATen/native/cuda/ForeachUnaryOp.cu

```diff
@@ -73,6 +73,7 @@ template <typename scalar_t, template<class> class Op> void foreach_unary_op_(Te
                                          /* r_args_depth */ 1,
                                          /* res_arg_index */ 0>(),
                           Op<opmath_t>());
+    maybe_increment_version(tensors);
```

Comment:
Do I misremember our discussion where you showed me an example where the version was bumped properly? Why does this need to be added?

Reply:
That happened when inputs tensors are a CPU tensor not when fast path is chosen.
Even the functions registered to native_functions.yaml with CUDA key could go into the slow path which just calls aten native function. Therefore I decided to put this manual version bump here

Reply:
Ho that's right!
I'm still not sure why the InplaceOrView kernel doesn't get generated automatically to do that already? But we can look into that later.

Reply:
I thought with https://github.com/pytorch/pytorch/blob/b33d9e2c87f9323e56672fe862fe1de704346a37/torchgen/model.py#L1388-L1405, inplace functions that modify (and return) multiple Tensors don't get caught by https://github.com/pytorch/pytorch/blob/9cf84347767c8abb8feba18a9a1baba321eeb8b9/tools/autograd/gen_inplace_or_view_type.py#L481-L482